File: src/main/java/org/motivepick/domain/ui/task/UpdateTaskRequest.java
package org.motivepick.domain.ui.task

import java.time.LocalDateTime

class UpdateTaskRequest {

    var name: String? = null
    var description: String? = null
    var created: LocalDateTime? = null
    var dueDate: LocalDateTime? = null
    var deleteDueDate: Boolean = false
    var closingDate: LocalDateTime? = null
    var closed: Boolean? = null
}

File: src/main/kotlin/org/motivepick/domain/ui/task/UpdateTaskRequest.kt
package org.motivepick.domain.ui.task

import java.time.LocalDateTime

class UpdateTaskRequest {

    var name: String? = null
    var description: String? = null
    var created: LocalDateTime? = null
    var dueDate: String? = null
    var closingDate: LocalDateTime? = null
    var closed: Boolean? = null
}
